### PR TITLE
Bump networking components for 1.8.9

### PR DESCRIPTION
**Pull request by GoelDeepak**
## High Level Description

This patch backports some of the mesos state parser fixes that many customers have reported.

## Related Issues

  - [DCOS_OSS-927](https://jira.mesosphere.com/browse/DCOS_OSS-927) Backport fix for mesos state parser to 1.8.x

## Checklist for all PR's

  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [x] Test Results: [link to CI job test results for component]
  - [x] Code Coverage (if available): [link to code coverage report]

https://github.com/dcos/minuteman/pull/104
https://github.com/dcos/minuteman/pull/109
https://github.com/dcos/navstar/pull/62

___
**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

## Instructions and Review Process

**What is the review process and when will my changes land?**

All PRs require 2 approvals using github's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).

Reviewers should be:
* Developers who understand the code being modified.
* Developers responsible for code that interacts with or depends on the code being modified.

It is best to proactively ask for 2 reviews by @mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the [next train](https://github.com/dcos/dcos/blob/master/contributing.md).

### CODE REVIEW

**Comment by GoelDeepak** [comment]
@mesosphere-mergebot label Work In Progress

**Comment by GoelDeepak** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by lingmann** [comment]
Hi @GoelDeepak can you please ping two people that would be good candidates to review this backport? One option is to take a look at whoever reviewed the original PR against master.

**Comment by GoelDeepak** [comment]
@lingmann request you to please add @drewkerrigan and/or @nlsun

File: packages/minuteman/build [hunk 1]
```diff
@@ -19,6 +19,7 @@ Description=Layer 4 Load Balancer: DC/OS Layer 4 Load Balancing Service
 Restart=always
 StartLimitInterval=0
 RestartSec=5
+LimitNOFILE=16384
```
Comment (nlsun) [inline packages/minuteman/build]: maybe also explain in the pr description why we are doing this since it's unrelated to the parser

**Comment by GoelDeepak** [comment]
@spahl @lingmann ship it?

**Comment by adam-mesos** [comment]
@GoelDeepak Can you create an EE PR off the 1.8.9 branch that bumps upstream to this branch, so we can see if the enterprise tests pass too?
Any idea if the azure and upgrade tests are expected to fail in 1.8.x?

**Comment by GoelDeepak** [comment]
@mesosphere-mergebot bump-ee

**Comment by GoelDeepak** [comment]
@mesosphere-mergebot bump-ee

**Comment by GoelDeepak** [comment]
@adam-mesos Both the failures (on-prem and enterprise) are independent of this PR.

**Comment by adam-mesos** [comment]
> If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath packages), then in addition to the above please also include:
> [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): example
> [ ] Test Results: [link to CI job test results for component]
> [ ] Code Coverage (if available): [link to code coverage report]

@GoelDeepak For each component you're bumping (navstar, minuteman) please provide links to the changelog/diff and test results. New/updated integration tests would be great too (how do we even know these backports fixed anything)? Just checking the boxes is not sufficient.

**Comment by GoelDeepak** [comment]
@adam-mesos provided requested information

**Pull request merged**